release notes/v0.47.0.md

```diff
@@ -0,0 +1,113 @@
+k6 `v0.47.0` is here 🎉! This release includes:
+
+## Deprecations
+
+- [#3347](https://github.com/grafana/k6/pull/3347) The built-in `statsd` output option has been deprecated, and users should use the [xk6-output-statsd](https://github.com/LeonAdato/xk6-output-statsd) extension instead. See [#2982](https://github.com/grafana/k6/issues/2982) for future plans.
+- [#3288](https://github.com/grafana/k6/pull/3288) Loading remote modules now requires users to prepend them with `https://`. Before, k6 would try to resolve importing remote modules by prepending `https://` if it was missing. This behavior has been deprecated and will be fully removed in the next release (v0.48.0).
+
+## New features
+
+### Add gRPC's binary metadata support [#3234](https://github.com/grafana/k6/pull/3234), [xk6-grpc#46](https://github.com/grafana/xk6-grpc/pull/46)
+
+The k6 gRPC modules (`k6/net/grpc` and `k6/experimental/grpc`) now support handling binary metadata that uses the `-bin` postfix, according to the gRPC specification.
+
+```js
+let resp = client.invoke("grpc.testing.TestService/EmptyCall", {}, { metadata: { "X-Load-Tester-bin": new Uint8Array([2, 200]) } })
+```
+
+Thanks to @sapphire-janrain for the contribution!
+
+### Add gRPC's reflection metadata support [#3343](https://github.com/grafana/k6/pull/3343), [xk6-grpc#46](https://github.com/grafana/xk6-grpc/pull/46)
+
+The k6 gRPC modules (`k6/net/grpc` and `k6/experimental/grpc`) now support adding metadata to reflection requests by using a new connection parameter [`reflectMetadata`](https://k6.io/docs/javascript-api/k6-net-grpc/client/client-connect/#connectparams).
+
+### Higher precision for Trend metrics in Cloud [#3302](https://github.com/grafana/k6/pull/3302)
+
+k6 Cloud is now able to store and visualize Trend metrics up to 3 digits of precision for decimal numbers.
+
+### Docker images for ARM64 architecture [#3320](https://github.com/grafana/k6/pull/3320)
+
+k6 now builds and pushes dedicated Docker images for ARM64. Check k6's [tags page](https://hub.docker.com/r/grafana/k6/tags) on DockerHub for details. 
```

Comment:
It's not really k6, so it's either neutral we or maybe k6's CI.

```suggestion
We started to build and push dedicated Docker images for ARM64. Check k6's [tags page](https://hub.docker.com/r/grafana/k6/tags) on DockerHub for details. 
```

Reply:
Applied,  not exactly the same but very similar.